src/transform-sdk/rust/core-types/src/lib.rs

```diff
@@ -116,13 +116,36 @@ impl<'a> From<&'a WrittenRecord<'a>> for BorrowedRecord<'a> {
     }
 }
 
+/// [`WriteOptions`] allows for customizing a [`RecordWriter`]'s write.
+///
+/// An example is to customize the output topic which a write can go to.
+#[derive(Default, PartialEq, Eq, Clone, Debug)]
+pub struct WriteOptions<'a> {
+    pub topic: Option<&'a str>,
+}
+
+impl<'a> WriteOptions<'a> {
+    /// Create a new options struct with the record destination to an optional `topic`.
```

Comment:
Is "record" here referring to `Record` specifically? And would "the record's destination" still be accurate?

Reply:
Yes. Updated PTAL